nautobot/users/filters.py

```diff
@@ -52,6 +88,19 @@ class Meta:
             "email",
             "is_staff",
             "is_active",
+            "config_data",
+            "changes",
+            "has_changes",
```

Comment:
If you have the field defined above it is not needed here. This section is for "magic" filters that introspect fields on the given model and provide determined proper filter and lookup expression combinations.